core/warning.ts

```diff
@@ -143,8 +143,8 @@ export class Warning extends Icon {
    */
   getText(): string {
     const allWarnings = [];
-    for (const id in this.text_) {
-      allWarnings.push(this.text_[id]);
+    for (const id in this.text) {
```

Comment:
is this for loop different than `this.text.values()`? we probably just didn't use it before because of IE? (i know you didn't add this but just wondering)